modules/transport/http/src/org/apache/axis2/transport/http/HTTPSender.java

```diff
@@ -256,6 +260,45 @@ private void sendViaPut(MessageContext msgContext, URL url,
         }
     }
 
+    /**
+     * Used to send a request via HTTP Patch Method
+     *
+     * @param msgContext - The MessageContext of the message
+     * @param url - The target URL
+     * @param soapActionString - The soapAction string of the request
+     * @throws AxisFault - Thrown in case an exception occurs
+     */
+    private void sendViaPatch(MessageContext msgContext, URL url,
+            String soapActionString) throws AxisFault {
+
+        HttpClient httpClient = getHttpClient(msgContext);
+        PatchMethod patchMethod = new PatchMethod();
+        MessageFormatter messageFormatter =
+                populateCommonProperties(msgContext, url, patchMethod, httpClient, soapActionString);
+
+        patchMethod.setRequestEntity(new AxisRequestEntity(messageFormatter,
+                msgContext, format, soapActionString,
+                chunked, isAllowedRetry));
+
+        if (!httpVersion.equals(HTTPConstants.HEADER_PROTOCOL_10) && chunked) {
+            patchMethod.setContentChunked(true);
+        }
+
+        String soapAction = messageFormatter.formatSOAPAction(msgContext, format, soapActionString);
+        if (soapAction != null && !msgContext.isDoingREST()) {
+            patchMethod.setRequestHeader(HTTPConstants.HEADER_SOAP_ACTION, soapAction);
+        }
+        try {
+            executeMethod(httpClient, msgContext, url, patchMethod);
+            handleResponse(msgContext, patchMethod);
+        } catch (IOException e) {
+            log.info("Unable to sendViaPatch to url[" + url + " ]", e);
```

Comment:
This need to be error message. But don't log and throw. Either log or throw.

Reply:
+1 Done